portfolio/src/main/webapp/indexES.html

```diff
@@ -12,12 +12,12 @@
     <div id="content">
         <section>
             <h1>Una breve introducci&oacute;n sobre m&iacute;</h1>
-            <img src= "1137.png" alt= "Yo en Cuidad de México" style = "width:240px; height:320px">
+            <img src= "images/presentation photo.png" alt= "Yo en Cuidad de México" style = "width:240px; height:320px">
```

Comment:
nit: prefer to avoid spaces in your file names. Whitespace can often lead to unexpected issues, it's usually better to use something like `presentation_photo.png`